libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1529,8 +1532,7 @@ void ExpressionCompiler::appendArithmeticOperatorCode(Token::Value _operator, Ty
 				exponent := result // Make accessible to outside world
```

Comment:
Ah, now I read the algorithm while being properly awake :)
I fear this won't do - the loop takes way too much time, we should use some square and multiply-like algorithm, but not sure if that is feasible for fixed points.